backend/start.sh

```diff
@@ -5,6 +5,7 @@ cd "$SCRIPT_DIR" || exit
 
 KEY_FILE=.webui_secret_key
 
+HOST="${HOST:-0.0.0.0}"
```

Comment:
In the interest of security (especially without any default transport encryption), shouldn't we limit the listener to 127.0.0.1 by default and require that the user override via `snapctl set host="0.0.0.0"` to listen on all addresses?